docs/source/changelog.md

```diff
@@ -6,6 +6,52 @@ command line for details.
 
 ## [Unreleased]
 
+## 2.0
+
+JupyterHub 2.0 is a big release!
+
+The most significant change is the addition of [roles and scopes][rbac]
+to the JupyterHub permissions model,
+allowing more fine-grained access control.
+Read more about it [in the docs][rbac].
+
+In particular, the 'admin' level of permissions should not be needed anymore,
+and you can now grant users and services only the permissions they need, not more.
+We encourage you to review permissions, especially any service or user with `admin: true`
+and consider assigning only the necessary roles and scopes.
```

Comment:
How does this relate to `admin_users`?

I recognize `admin: true` from `c.JupyterHub.services` but not from c.Authenticator.admin_users` for example.

Reply:
Ah, forgot about that. I'll do one more pass on references to admin_users.

Reply:
updated some notes on admin_users to point folks toward roles and mention roles as preferable for limiting permissions.